app.py

```diff
@@ -0,0 +1,40 @@
+
+from opentok import OpenTok
+from flask import Flask, request
+app = Flask(__name__)
+app.config.from_pyfile('.env')
+
+opentok = OpenTok(app.config.get('OPENTOK_API_KEY'), app.config.get('OPENTOK_API_SECRET'))
+
+
+@app.route('/')
+def index():
+    return "Success! Endpoints available: /session."
+
+
+@app.route('/session', methods=['POST'])
+def session():
+    # return "Success! Endpoints available: /session."
+    error = None
+    args = []
+    if request.method == 'POST':
+        if request.form['location']:
```

Comment:
Are these required parameters? Because I was unable to make a cURL POST request without them:
```
 curl -X POST localhost:5000/sessio
```

``` 
File "/Users/bgreenberg2/Documents/dev/python/aws/opentok-session-lambda-python/app.py", line 21, in session
    if request.form['location']:
  File "/Users/bgreenberg2/Documents/dev/python/aws/opentok-session-lambda-python/venv/lib/python3.8/site-packages/werkzeug/datastructures.py", line 442, in __getitem__
    raise exceptions.BadRequestKeyError(key)
werkzeug.exceptions.BadRequestKeyError: 400 Bad Request: The browser (or proxy) sent a request that this server could not understand.
KeyError: 'location'
```

Reply:
Fixed. Set this to use a dictionary, and properly set enums. No longer erroring.